### PR TITLE
Add nobump version add usage message

### PR DESCRIPTION
**Pull request by legalsylvain**

### CODE REVIEW

File: src/oca_github_bot/commands.py [hunk 1]
```diff
@@ -51,11 +58,30 @@ def __init__(self, name, options):
...
-            return BotCommandMerge(name, options)
-        else:
-            raise InvalidCommandError(name)
+    def create(cls, name, options, github_context):
+        try:
+            if name == "merge":
+                return BotCommandMerge(name, options)
+            else:
+                raise InvalidCommandError(name)
+        except CommandError as e:
```
Comment (sbidoul) [inline src/oca_github_bot/commands.py]: I like the CommandError exception hierarchy.

However I would rather move this error handling to the webhook, so the BotComand class need to be concerned about the context in which it was called (could be PR but also an issue, and why not something else such a CLI in the future).

File: src/oca_github_bot/config.py [hunk 2]
```diff
@@ -92,3 +92,13 @@ def func_wrapper(*args, **kwargs):
...
+
+OCABOT_USAGE = (
+    "**Ocabot commands**\n"
+    "* ``ocabot merge major|minor|patch|nobump``\n\n"
+    "**More information**\n"
+    " * [ocabot documentation](https://github.com/OCA/oca-github-bot/#commands)\n"
+    " * [OCA guidelines](https://github.com/OCA/odoo-community.org/blob/master/"
+    "website/Contribution/CONTRIBUTING.rst), "
+    'specially the "Version Numbers" section.'
+)
```
Comment (sbidoul) [inline src/oca_github_bot/config.py:107]: At some point we'll probably want to make this configurable too.

File: src/oca_github_bot/commands.py [hunk 3]
```diff
@@ -51,11 +58,30 @@ def __init__(self, name, options):
...
+                    github_context.get("repo", False),
+                    github_context.get("pr", False),
+                )
+                options_text = " ".join(options)
+                user_text = github_context.get("username", "")
+                github.gh_call(
+                    gh_pr.create_comment,
+                    f"Hi @{user_text}."
+                    f"Your command ``{name} {options_text}`` failed.\n\n"
+                    f"``Error : ``\n"
```
Comment (sbidoul) [inline src/oca_github_bot/commands.py]:
```suggestion
```

File: src/oca_github_bot/commands.py (hunk 1 above)
Comment (legalsylvain) [reply src/oca_github_bot/commands.py]: Thanks for your advice. I moved in the webhook. Indeed, I was not very confortable with the current design.
Furthermore, I had to add a context=None, in the end of some function, and it reminded me of the dark hours of openerp. ;-)
Could you check if it's ok now ?

File: src/oca_github_bot/config.py (hunk 2 above)
Comment (legalsylvain) [reply src/oca_github_bot/config.py:107]: Done. 

In fact, splitted into 2 parameters. a some generic one OCABOT_USAGE, and another one OCABOT_EXTRA_DOCUMENTATION for the links, etc.

File: src/oca_github_bot/commands.py (hunk 3 above)
Comment (legalsylvain) [reply src/oca_github_bot/commands.py]: Done.

**Comment by legalsylvain** [comment]
@sbidoul 

I updated the code, and change some minors things in the comment of the bot. 

Tested again here : https://github.com/grap/github-ocabot-test/pull/11

**Pull request merged**